app/src/components/domain/RiskImminentEvents/Gdacs/index.tsx

```diff
@@ -178,6 +286,12 @@ function Gdacs(props: Props) {
             activeEventExposure={exposureResponse}
             activeEventExposurePending={exposureResponsePending}
             onActiveEventChange={handleActiveEventChange}
+            activeLayersMapping={activeLayersMapping}
+            layers={layers}
+            onLayerChange={handleLayerChange}
+            // NOTE: return type void function cannot be optional
+            // popup is not need in gdacs cyclone track points
+            handlePopupClose={noOp}
```

Comment:
Why not just make this optional? It should be possible to make this prop optional.